webui/react/src/pages/JobQueue/JobQueue.table.tsx

```diff
@@ -101,6 +103,7 @@ export const columns: ColumnDef<Job>[] = [
     key: 'name',
     render: (_: unknown, record: Job): ReactNode => {
       let label: ReactNode = null;
+      if (!('name' in record)) return OMITTED_STR;
```

Comment:
**super nit:** swap the `let label` and this line since we won't need that label variable if we are rendering OMITTED_STR